ui/raidboss/data/triggers/o5s.js

```diff
@@ -58,49 +58,37 @@
   ],
   triggers: [
     {
-      regex: /04:Removing combatant Phantom Train/,
-      regexDe: /04:Removing combatant Phantomzug/,
-      regexFr: /04:Removing combatant Train Fantôme/,
-      regexJp: /04:Removing combatant 魔列車/,
+      regex: /04:Removing combatant (?:魔列車|Phantom Train|Phantomzug|Train Fantôme)/,
       run: function(data) {
         data.StopCombat();
       },
     },
     {
       id: 'O5S Doom Strike',
-      regex: /14:28B1:Phantom Train starts using Doom Strike on (\y{Name})/,
-      regexDe: /14:28B1:Phantomzug starts using Vernichtungsschlag on (\y{Name})/,
-      regexFr: /14:28B1:Train Fantôme starts using Frappe Létale on (\y{Name})/,
-      regexJp: /14:28B1:魔列車 starts using 魔霊撃 on (\y{Name})/,
+      regex: /14:28B1:(?:魔列車|Phantom Train|Phantomzug|Train Fantôme) starts using (?:魔霊撃|Doom Strike|Vernichtungsschlag|Frappe Létale) on (\y{Name})/,
```

Comment:
This undos what I did in 4022e58.  I intentionally want to have different regexes for different languages here.  I think it makes regexes more readable, makes cactbot less complicated to not have to handle two language settings elsewhere, and also makes it possible to write automated tools to know what translations are missing.  I just don't think folks using different languages for the ACT plugin vs the ffxiv games is something I want to support.

(It looks like you had your own local version of the file that and didn't make changes based on the latest version on github, so you have a lot more changes in this file than I think there should be.)